metaflow/plugins/argo/argo_client.py

```diff
@@ -87,6 +87,44 @@ def register_workflow_template(self, name, workflow_template):
                 json.loads(e.body)["message"] if e.body is not None else e.reason
             )
 
+    def delete_workflow_schedule(self, name):
+        client = self._kubernetes_client.get()
+
+        try:
+            return client.CustomObjectsApi().delete_namespaced_custom_object(
+                group=self._group,
+                version=self._version,
+                namespace=self._namespace,
+                plural="cronworkflows",
+                name=name,
+            )
+        except client.rest.ApiException as e:
+            if e.status == 404:
+                return None
+            else:
+                raise ArgoClientException(
+                    json.loads(e.body)["message"] if e.body is not None else e.reason
+                )
+
+    def delete_workflow_template(self, name):
+        client = self._kubernetes_client.get()
+
+        try:
+            return client.CustomObjectsApi().delete_namespaced_custom_object(
+                group=self._group,
+                version=self._version,
+                namespace=self._namespace,
+                plural="workflowtemplates",
+                name=name,
+            )
+        except client.rest.ApiException as e:
+            if e.status == 404:
+                return None
```

Comment:
I can see that later, `None` shall be interpreted as "not found", and handled by caller.  May be we should document this somewhat at the top of the function, to make it clear to future developers?